src/api/main.py

```diff
@@ -196,6 +197,7 @@
     initialize_templates()
     initialize_nonhumans()
     populate_stakeholder_shortname()
+    populate_cycle_tasks
```

Comment:
should this be a function call?

Reply:
Yes! My mistake. Resolved with [f6b33a6](https://github.com/cisagov/con-pca-api/pull/839/commits/f6b33a66119c06a41763e8a4f9df2a851c9a3ca3)